README.md

```diff
@@ -1 +1,56 @@
-# ReactChamps
+# Stat Check
+
+[Lien du Trello](https://trello.com/b/uc8WwvZB/stat-check)
+
+## Goal
+This project is a way for me to practice and perfect my React programming skills. It is a learning tool that allows me to experiment and try various things out when it comes to front-end development. It also allows me to discover new technologies and learn to master them, all based around a theme that personnaly interests me : video games, and particularly League of Legends.
```

Comment:
It also has a purpose to learn to work on agile method

Reply:
Yes, I forgot to mention the organization and mostly time-management aspect of the learning process.